simpleui-server/src/props-file-reader.ts

```diff
@@ -18,23 +18,56 @@ export class VersionProps {
 export class PropsFileReader {
 
     static PropertySets = [];
+    static cmdVars: CommandArgs = <CommandArgs> {
+        valid: true,
+        help: '\nSyntax:\n\n', // Each arg help is appended in arg definition
+        examples: '\n\nExample for running as a daemon:' +
+            '\n  node simpleui-server.js \\' +
+            '\n     --appName=bms \\' +
+            '\n     --webPort=2080' +
+
+            '\n\nExample for running the test:' +
+            '\n  node simpleui-server.js \\' +
+            '\n    --mode=test \\' +
+            '\n    --appName=bms \\' +
+            '\n    --webPort=2080 \\' +
+            '\n    --xmlInFile=src/public/simpleui-server/test/bms.reference.xml \\' +
+            '\n    --jsonOutFile=src/public/simpleui-server/test/bms.reference.json',
+
+        errors: 'Invalid command line:\n\n', // Each error is appended when parsing args
+        mode: 'daemon',
+        appName: '',
+        webPort: '2080',
+        xmlInFile: '',
+        jsonOutFile: '',
+        zmqHostname: '',
+        dbName: '',
+        themeName: '',
+    };
+
+    static getAppPropsFiles(): any {
+        let rawAppNameList = PropsFileReader.cmdVars.appName;
+        let webPort = PropsFileReader.cmdVars.webPort;
 
-    static getAppPropsFiles(rawAppNameList: string, webPort: string): any {
         const appPropsTuples = [];
         const appNames = rawAppNameList.split(/[,:;]/);
         for (let appName of appNames) {
             appName = appName.trim();
-            let isMock = (appName.substr(0, 5) === "mock-");
+
+            let isMock = (appName.substring(0, 5) === "mock-");
```

Comment:
should probably add a separate `--mock` flag to the command line, doing stuff like this basically requires you to read all the code to figure out how to use it

Reply:
From what I can tell, I think the `mock` option might be dead code. It seems like it allows you to give simpleui to a different directory that isnt `/var/www/APP_NAME`. I think this would be solely to have simpleui server serve `mock` data to the frontend, which is already done in the dev environment. I'll make sure of this and will remove the `mock` option if I am correct